dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -601,15 +665,15 @@ protected void deleteChildren(String inode) {
 			try {
 				conn.rollback();
 			} catch (SQLException e1) {
-				e1.printStackTrace();
+				Logger.error(CategoryFactoryImpl.class,e1);
 			}
-			e.printStackTrace();
+			Logger.error(CategoryFactoryImpl.class, e);
 		} finally {
 			try {
 				s.close();

```

Comment:
you can use CloseUtils.closeQuietly(s, conn)
Also set better names such as statement and connection

Reply:
Nice piece of utility. done